C/catchaMouse16/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "catchaMouse16"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/DynamicsAndNeuralSystems/catchaMouse16.git",
+              "2a952451acf6114de562da6007e37ff6e013b157")
+]
+
+# Bash recipe for building across all platforms
+makefile = raw"""
+CC = gcc
+CFLAGS = -std=c11 -fPIC -Wall -Wextra -g -O2 -lm -lgsl -lgslcblas
+LDFLAGS = -shared -lm -lgsl -lgslcblas
+RM = rm -f
+TARGET_LIB = "lib$(SRC_NAME).$(dlext)"
+
+SRCS = main.c fft.c stats.c helper_functions.c histcounts.c CO_AddNoise.c CO_AutoCorr.c CO_HistogramAMI.c CO_NonlinearAutocorr.c CO_TranslateShape.c DN_RemovePoints.c FC_LoopLocalSimple.c IN_AutoMutualInfoStats.c PH_Walker.c SC_FluctAnal.c ST_LocalExtrema.c SY_DriftingMean.c SY_SlidingWindow.c
```

Comment:
Maybe do https://github.com/DynamicsAndNeuralSystems/catchaMouse16/blob/56ad41b669a750df3be60d008010c8a84a5cf4bd/C/Makefile#L18 instead of listing the files manually?  This isn't very sustainable if the source code upstream changes.